release-notes.md

```diff
@@ -0,0 +1,64 @@
+# v0.1.0
+
+This release has been tracked through the [0.1.0 project](https://github.com/intel/cloud-hypervisor/projects/1).
+
+Highlights for `cloud-hypervisor` version 0.1.0 include:
+
+### Shared filesystem
+
+We added support for the [virtio-fs](https://virtio-fs.gitlab.io/) shared file
+system, allowing for an efficient and reliable way of sharing a filesystem
+between the host and the `cloud-hypervisor` guest.
```

Comment:
Should we mention that it is not a complete support of virtio-fs? I mean to say that we don't support the shared memory region which is DAXable and allow for very good performances.

Reply:
Maybe we should have that in the `docs/fs.md` file instead?

Reply:
Yes that'd be better indeed.